templates/expo-template-bare-minimum/ios/HelloWorld/AppDelegate.m

```diff
@@ -63,14 +63,17 @@ - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge {
 
 // Linking API
 - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
-  return [RCTLinkingManager application:application openURL:url options:options];
+  return [super application:application openURL:url options:options] || [RCTLinkingManager application:application openURL:url options:options];
 }
 
 // Universal Links
 - (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(nonnull void (^)(NSArray<id<UIUserActivityRestoring>> * _Nullable))restorationHandler {
-  return [RCTLinkingManager application:application
-                   continueUserActivity:userActivity
-                     restorationHandler:restorationHandler];
+  return [super application:application
+       continueUserActivity:userActivity
+         restorationHandler:restorationHandler]
+  || [RCTLinkingManager application:application
+               continueUserActivity:userActivity
+                 restorationHandler:restorationHandler];
 }
```

Comment:
i'm always confused by the return value or sequence to process the handlers. but in this case, maybe it's better to go through `RCTLinkManager`. that would not break react-native universal linking.

Reply:
Since the `RCTLinkingManager` always returns true, we can simply ignore returning it for now.

Reply:
hey @EvanBacon, but EXLegacyAppDelegateWrapper's methods didn't return true as default. So, your commit can cause the methods always return false.